test/e2e/specs/multi-block-selection.test.js

```diff
@@ -77,4 +77,44 @@ describe( 'Multi-block selection', () => {
 		await pressWithModifier( META_KEY, 'a' );
 		await expectMultiSelected( blocks, true );
 	} );
+
+	it( 'Should select/unselect multiple blocks using Shift + Arrows', async () => {
+		const firstBlockSelector = '[data-type="core/paragraph"]';
```

Comment:
The actual issue was for 3 blocks of the same type. We should another test :)

Reply:
The issue for me was happening no matter the type of the blocks.